### ISSUE TITLE
Scheduler doesn't work with capital S in element name

### ISSUE THREAD

**Issue by jacodg** [ibissource/iaf#2077]
**Describe the issue**
When using \<Scheduler\> instead of \<scheduler\> in a configuration file the scheduler doesn't work and doesn't show up in the scheduler page of the Frank!Console. All other elements in my configuration use capital letters and everything works but for the scheduler to work I have to change the first letter of the element name to lower case.

**Reporter**
Jaco de Groot

**To Reproduce**
Use the configuration below and see that the scheduler doesn't work. Change the \<Scheduler\> to \<scheduler\> (and the closing element too) and see that after this change it works.

**Screenshots**
\-

**Configuration**
```
<Configuration name="Povis demo">
	<Adapter name="Update temperature" description="Get current temperature in Amsterdam from from OpenWeather every 10 minutes and post it to ThingsBoard">
		<Receiver name="Update temperature">
			<JavaListener name="Update temperature"/>
		</Receiver>
		<Pipeline>
			<FixedResultPipe name="Discard input" returnString=""/>
			<!-- ... skipped ... -->
			<Exit path="EXIT" state="success"/>
		</Pipeline>
	</Adapter>
	<Scheduler>
		<Job
			description="Update temperature every 10 minutes"
			name="Update temperature"
			adapterName="Update temperature"
			receiverName="Update temperature"
			function="sendMessage"
			interval="600000"
		/>
	</Scheduler>
</Configuration>
```

**Input**
No input required

**Environment**
```
FF! 7.7-20210719.163234: Frank2Povis
Running on LPZWNL000040359 using Apache Tomcat/9.0.50
Java Version: OpenJDK Runtime Environment (1.8.0_292-b10)
Heap size: 258M, total JVM memory: 721M
Free disk space: 196GB, total disk space: 475GB
Up since: 2021-07-28 14:34:30 (7m) 
```

**Additional Environment**
\-

**Additional Context**
\-

**Comment by ricardovh** [comment]
I think the problem is that the Scheduler-element is not in https://github.com/ibissource/iaf/blob/master/core/src/main/resources/xml/xsl/uglify_lookup.xml. Related to #1350?

**Comment by alisihab** [comment]
Seems to be fixed with #2120.
